docs/dev/src/ci.md

```diff
@@ -19,3 +19,25 @@ To run `e2e-test` and `e2e-source-test` for `main-cron` in your pull request:
 1. Add `ci/run-e2e-test`.
 2. Add `ci/run-e2e-source-tests`.
 3. Add `ci/main-cron/run-selected` to skip all other steps which were not selected with `ci/run-xxx`.
+
+## Main Cron Bisect Guide
+
+1. Create a new build via buildkite: https://buildkite.com/risingwavelabs/main-cron/builds/#new
+2. Add the following environment variables:
+   - `START_COMMIT`: The lower bound (inclusive) of the regression range.
+   - `END_COMMIT`: The commit hash of the bad commit we observed.
```

Comment:
Is it inclusive?

Reply:
Exclusive

Reply:
Hmm.. Not sure if I understand it correctly. What if "the bad commit we observed" is exactly the first one that causes problems? 🤔

Reply:
I meant exclusive in the sense it won't be tested again. But that's a little confusing.

I think first `good` commit and `bad` commit is a better way to phrase it. Updated the docs.